src/gt4py/testing/suites.py

```diff
@@ -502,12 +528,14 @@ def test_implementation(self, test, parameters_dict):
                 name: tuple(
                     nb[0] - g[0] for nb, g in zip(new_boundary, cls.symbols[name].boundary)
                 )
-                for name in implementation.field_info.keys()
+                for name in inputs
+                if name in implementation.field_info
```

Comment:
Question: why is the iteration loop reversed compared with the previous version? I mean, why is not something like:

```suggestion
                for name in implementation.field_info.keys()
                if name in inputs
```

Reply:
I did so for coherency with other loops already present. Plus it makes clearer to me that we iterate over all inputs, disregarding anything which is not used. But I think it's a matter of taste, so I'm open to revert this change.

Reply:
Fine for me, I think it's just a matter of taste.